salt/modules/boto_lambda.py

```diff
@@ -200,6 +200,7 @@ def _filedata(infile):
 def create_function(FunctionName, Runtime, Role, Handler, ZipFile=None,
                     S3Bucket=None, S3Key=None, S3ObjectVersion=None,
                     Description="", Timeout=3, MemorySize=128, Publish=False,
+                    VpcConfig=None,
```

Comment:
Can you move this new kwarg to the end of the kwarg definition list in this function? That way we can avoid an API change and any potential regressions if people are using positional kwargs.